machineconfiguration/v1/0000_80_machineconfigpool.crd.yaml

```diff
@@ -87,9 +87,6 @@ spec:
             spec:
               description: MachineConfigPoolSpec is the spec for MachineConfigPool resource.
               type: object
-              required:
-                - configuration
```

Comment:
this is currently not required in the MCO: https://github.com/openshift/machine-config-operator/blob/40571adccaa3ef49a823abe8d7494bcdc48f2021/install/0000_80_machine-config-operator_01_machineconfigpool.crd.yaml#L93C13-L93C13